### PR TITLE
Allow to add multiple uid as criteria for classcontent::getCollection

### PR DESCRIPTION
**Pull request by fkroockmann**
Now we can :

/res/1/classcontent?uids=foo,bar

### CODE REVIEW

**Comment by crouillon** [comment]
:+1:

File: ClassContent/Repository/ClassContentQueryBuilder.php [hunk 1]
```diff
@@ -77,9 +77,12 @@ public function addSiteFilter($site)
         if ($site instanceof Site) {
             $site = $site->getUid();
         }
-        $this->andWhere(
-            'cc._uid IN (SELECT i.content_uid FROM BackBee\ClassContent\Indexes\IdxSiteContent i WHERE i.site_uid = :site_uid)'
-        )->setParameter('site_uid', $site);
+
+        if (!empty($site)) {
```
Comment (eric-chau) [inline ClassContent/Repository/ClassContentQueryBuilder.php]: To keep things simple, can you use an extra variable? Something like that:

```php
$uid = null;
if ($site instanceof Site) {
    $uid = $site->getUid();
}

if (null !== $uid) {
    // do your stuff
}
```

Comment (fkroockmann) [inline ClassContent/Repository/ClassContentQueryBuilder.php]: This function accept uid or instance of Site\Site

File: Rest/Controller/ClassContentController.php [hunk 2]
```diff
@@ -652,6 +652,10 @@ private function findContentsByCriteria(array $classnames, $start, $count)
         unset($criterias['order_by']);
         unset($criterias['order_direction']);
 
+        $criterias['contentIds'] = $this->computeUids();
```
Comment (eric-chau) [inline Rest/Controller/ClassContentController.php]: I don't think that you need to do it in a function. What do you think about something like that? 
```php
$criterias['contentIds'] = explode(',', $this->getRequest()->query->get('uids', ''));
```
So you can remove ``::computeUids()``.

File: ClassContent/Repository/ClassContentQueryBuilder.php (hunk 1 above)
Comment (eric-chau) [inline ClassContent/Repository/ClassContentQueryBuilder.php]: oh ok. Nevermind then.

**Comment by eric-chau** [comment]
:+1:

File: Rest/Controller/ClassContentController.php (hunk 2 above)
Comment (fkroockmann) [inline Rest/Controller/ClassContentController.php]: Right :)

**Pull request merged**